sources/org.osbuild.files

```diff
@@ -71,8 +83,18 @@ def main(options, checksums, cache, output):
 
     with concurrent.futures.ProcessPoolExecutor(max_workers=15) as executor:
         requested_urls = []
+        rhsm_secrets = None
+
         for checksum in checksums:
             try:
+                secrets = urls[checksum].get("secrets")
+                # if the secret is for rhsm replace the secrets value with the rhsm certs
+                if secrets == "org.osbuild.rhsm":
+                    if rhsm_secrets is None:
+                        rhsm_command = ["python3", "secrets/org.osbuild.rhsm"]
+                        rhsm_secrets = json.loads(subprocess.run(rhsm_command, stdout=subprocess.PIPE).stdout)
+                    urls[checksum]["secrets"] = rhsm_secrets
+
```

Comment:
I think extracting this into its own small function makes the code more readable.

Reply:
I didn't see the benefit of this because only 3 lines of code would be moved to a separate function